tensorflow/python/ops/image_ops_impl.py

```diff
@@ -464,6 +464,13 @@ def flip_left_right(image):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+    
+  Usage Example:
+    ```
+    >>> image = tf.random.normal(shape=(256, 256, 3))
+    >>> tf.image.flip_left_right(image)
+    <tf.Tensor 'flip_left_right/ReverseV2:0' shape=(256, 256, 3) dtype=float32>
+    ```
```

Comment:
Hi, by following the guide you should remove the triple backticks because it's not needed for the doctest. Also indent the code as the same level as "Usage Example:"

JuanC

Reply:
You can refer to the guide here https://www.tensorflow.org/community/contribute/docs_ref

Reply:
You can refer to the guide here https://www.tensorflow.org/community/contribute/docs_ref